docs/insomnia/git-sync.md

```diff
@@ -6,74 +6,75 @@ category-url: get-started
 
-When configuring a remote repository, you will be prompted for the following information. 
+When configuring a remote repository, you will be prompted for the following information.
 
-* **Git URI**: The URI of the git repository you wish to connect to. Note, only https URLs are supported.
-* **Author Name/Email**: Git author metadata that will be stored with each commit
-* **Authentication Token**: The token needed to authenticate with remote repository provider (GitHub, Bintray, etc). If you have two-factor authentication (2FA) enabled on your account, it is unlikely you will be able to use your username and password. Instead, generate a Personal Access Token (or an App Password in Bitbucket lingo). 
+* **Git URI (https)**: The URI of the Git repository. Only HTTPS URLs are supported.
+* **Author Name**: The Git author name to store with each commit.
+* **Author Email**: The Git author email to store with each commit.
+* **Username**: The Git author username to match with the authentication token.
+* **Authentication Token**: The token needed to authenticate with remote repository provider, such as GitHub or BitBucket. If you have two-factor authentication (2FA) enabled on your account, it is unlikely you will be able to use your username and password. Instead, generate a Personal Access Token or App Password (see list below for links to documentation for your Git system).
 
 Find instructions on how to create a Personal Access Token on the following platforms:
 
 * [Github](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+* [Gitlab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
 * [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords/)
 * [Bitbucket Server](https://confluence.atlassian.com/bitbucketserver/personal-access-tokens-939515499.html)
-* [Gitlab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)
 
-Once complete, click **Done** and the repository settings will be persisted for future operations. Author details and token can be updated after if needed.
+Once complete, click **Done** and the repository settings will be persisted for future operations. Author details and token can be updated as needed.
 
 ## Clone an Existing Repository
 
-If a team member has already pushed a Collection to a remote repository, it can be cloned via the main menu in the top-right of the application on the Dashboard view. Here, you will see the same Repository Settings dialog to configure remote access.
+If a team member has already pushed a Document to a remote repository, it can be cloned via the main menu in the top-right of the application on the Dashboard view. Here, you will see the same Repository Settings dialog to configure remote access.
 
 {:.alert .alert-primary}
 **Note**: Insomnia does not currently support repositories that contain files outside the root **.insomnia** folder.
```

Comment:
Funky, this must be from the very first iteration. It _does_ support files outside of the root `.insomnia` folder, it just doesn't consume them (eg. if you have code and your `.insomnia` folder alongside in the same repository, that's fine).

This alert can be removed 🎉